src/Sylius/Behat/Context/Api/Admin/ManagingPromotionsContext.php

```diff
@@ -45,13 +62,24 @@ public function iWantToCreateANewPromotion(): void
         $this->client->buildCreateRequest(Resources::PROMOTIONS);
     }
 
+    /**
+     * @When I want to modify a :promotion promotion
+     * @When /^I want to modify (this promotion)$/
+     * @When I modify a :promotion promotion
+     */
+    public function iWantToModifyAPromotion(PromotionInterface $promotion): void
+    {
+        $this->client->buildUpdateRequest(Resources::PROMOTIONS, $promotion->getCode());
+    }
+
     /**
      * @When I specify its :field as :value
+     * @When I do not specify its :field
      * @When I :field it :value
      */
-    public function iSpecifyItsAs(string $field, string $value): void
+    public function iSpecifyItsAs(string $field, ?string $value = null): void
     {
-        $this->client->addRequestData($field, $value);
+        $this->client->addRequestData($field, (string) $value);
```

Comment:
I would be for not casting the value to string or simply omit setting this value in request if it is null